docs/api-designer/getting-started-api-designer/README.adoc

```diff
@@ -101,147 +108,142 @@ In this quick start, you'll learn how to ...
 endif::[]
 
 [id="proc-creating-schema-api-design_{context}"]
-== Creating an {product-api-designer} schema or API design
+== Creating a schema or API design with {product-long-api-designer}
 
 [role="_abstract"]
-Use the {product-long-api-designer} web console to create an {product-api-designer} schema or API design.
+Use the {product-long-api-designer} web console to create an event schema or API definition.
 
 ifndef::qs[]
 .Prerequisites
 * You're logged in to the {product-api-designer} web console at {service-url-api-designer}[^].
 endif::[]
 
 .Procedure
-. In the {service-url-api-designer}[{product-api-designer} web console], click ...
-. Click ...
+. In the {service-url-api-designer}[{product-api-designer} web console], click *Create a schema or API design*.
+. Complete the form to provide the details of the new design:
+.. *Name*: Enter a unique design name, such as `my-api-design`.
+.. *Summary*: Optional: Enter a short description of the design.
+.. *Type*: Accept the default value: *OpenAPI*.
+.. *Version*: Accept the default value: *3.0.2*.
+.. *Template*: Select the *Pet Store Example* template.
 +
 [.screencapture]
 .Creating an {product-api-designer} design
 image::create-api-designer-design.png[Image of page to create an {product-api-designer} design]
-. Click ...
-
-.Verification
-ifdef::qs[]
-* Is the new {product-api-designer} design listed in the designs table?
-* Is ...
-endif::[]
-ifndef::qs[]
-. Verify that the new {product-api-designer} design is listed in the designs table.
-. Verify that ...
-endif::[]
-
-[id="proc-importing-schema-api-design_{context}"]
-== Importing an {product-api-designer} schema or API design
-
-[role="_abstract"]
-You can import an {product-api-designer} schema or API design from a URL, from a file, or from an {product-long-registry} instance.
-
-.Prerequisites
-* You're logged in to the {product-api-designer} web console at {service-url-api-designer}[^].
-
-.Procedure
-. In the *{product-api-designer}* designs page of the web console, click ...
-. Click ...
-
-.Verification
-ifdef::qs[]
-* Is the imported {product-api-designer} design listed in the designs table?
-* Is ...
-endif::[]
-ifndef::qs[]
-. Verify that the imported {product-api-designer} design is listed in the designs table.
-. Verify that ...
-endif::[]
++
+. Click *Create* to create the design. The design editor opens automatically.
```

Comment:
Do we state anywhere that the design editor is only available for certain types? (e.g., not available for Protobuf)

Reply:
OK, cool. Might do no harm to add something like "(OpenAPI and AsyncAPI only)" after "Click the *Design* tab" anyway so users creating schemas can skip it? I wasted a bit of time looking for the Design tab for my Protobuf schema :)

Reply:
Apologies, I misread your question! As discussed in chat, I'll add a section about creating a schema design (no design editor).

Reply:
Added in latest commit :)